src/renderers/dom/shared/__tests__/ReactMount-test.js

```diff
@@ -297,18 +295,6 @@ describe('ReactMount', () => {
     expect(calls).toBe(5);
   });
 
-  it('tracks root instances', () => {
-    // Used by devtools.
-    expect(Object.keys(ReactMount._instancesByReactRootID).length).toBe(0);
-    ReactTestUtils.renderIntoDocument(<span />);
-    expect(Object.keys(ReactMount._instancesByReactRootID).length).toBe(1);
-    var container = document.createElement('div');
-    ReactDOM.render(<span />, container);
-    expect(Object.keys(ReactMount._instancesByReactRootID).length).toBe(2);
-    ReactDOM.unmountComponentAtNode(container);
-    expect(Object.keys(ReactMount._instancesByReactRootID).length).toBe(1);
-  });
-
```

Comment:
@gaearon  I wonder if we should have a test that mocks`__REACT_DEVTOOLS_GLOBAL_HOOK__` and then runs a copy of https://github.com/facebook/react-devtools/blob/master/backend/attachRenderer.js#L24 against it?

The purpose of this test is to ensure that we don't accidentally remove these hooks because they look unused.